messaging-experiments.json

```diff
@@ -1629,5 +1629,250 @@
       "experimentDocumentUrl": "https://experimenter.services.mozilla.com/experiments/message-router-content-experiment-high-velocity-message-testing-cfr-to-pip/",
       "userFacingDescription": "Message Router Content Experiment: High Velocity Message Testing: CFR to PiP"
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V3-80-RELEASE",
+    "enabled": true,
+    "filter_expression": "env.version >= '80.' && env.version < '81.' && env.channel == 'beta'",
```

Comment:
changing filterexpression to use default channel will help verify experiment with local build

"filter_expression": "env.version >= '80.' && env.version < '81.' && (env.channel == 'beta' || env.channel == 'default')"

Reply:
done!